pdns/ixfrdist.cc

```diff
@@ -707,6 +710,7 @@ int main(int argc, char** argv) {
       ("server-address", po::value<string>()->default_value("127.0.0.1:5300"), "server address")
       ("work-dir", po::value<string>()->default_value("."), "Directory for storing AXFR and IXFR data")
       ("keep", po::value<uint16_t>()->default_value(KEEP_DEFAULT), "Number of old zone versions to retain")
+      ("axfr-timeout", po::value<uint16_t>()->default_value(AXFRTIMEOUT_DEFAULT), "Timeout in seconds for an AXFR to complete")
```

Comment:
It looks like it configures the timeout for getting a chunk of the `AXFR`, not for the whole `AXFR`, right?

Reply:
Correct, the wording is a bit off...

Reply:
One option would be to keep track of the total elapsed time in `AXFRRetriever`, as we do for the number of bytes, but I'm not sure if that's you have in mind?

Reply:
That could also work, but that should be initialized on the first `getChunk` and be checked every consecutive call to `getChunk`. It also does not feel right.